clang/Cursor.d

```diff
@@ -281,6 +281,11 @@ struct Cursor
         foreach (cursor; unit.cursor.children)
             result[cursor.spelling] = cursor.kind;
 
+        auto version_ = clangVersion();
+
+        if (version_.major == 3 && version_.minor == 7)
+            result["__int64"] = CXCursorKind.CXCursor_MacroDefinition;
```

Comment:
Version 3.7 doesn't recognize __int64 and it is used in MinGW. It adds a macrodefinition for __int64 as long long.